IFC 4.3/Earthworks_01/UT_Earthworks_01.ifc

```diff
@@ -0,0 +1,65 @@
+#38= IFCGEOGRAPHICELEMENT('071iuxsZ189wc6zewS91Ly',#2,$,$,$,#25,#39,$,.TERRAIN.);

```

Comment:
In IFC4x3_RC2, `IfcGeographicElement` is deprecated (not noted in documentation, but in its TypeEnum). Use `IfcGeoModel` with `IfcSolidStratum`. Representation with TIN can stay.

Reply:
CORRECTION @pjanck 
IfcGeoModel is a geotechnical Assembly and is in essence the same as IfcElementAssembly in it is a aggregator, if the terrain is a singular surface model it should be IfcSolidStratum, if there are multiple layers defined then each layer is a IfcXStratum and grouped into the IfcGeoModel.

Reply:
Thank you @AlexBrad1ey for the correction! I agree wholehaeartedly.

Reply:
I replaced IfcGeographicElement with IfcSolidStratum, but I don't know where/how to use IfcElementAssembly?